xwiki-rendering-api/src/main/java/org/xwiki/rendering/block/NumberedListBlock.java

```diff
@@ -31,7 +31,7 @@
  * @version $Id$
  * @since 1.5M2
  */
-public class NumberedListBlock extends AbstractBlock implements ListBLock
+public class NumberedListBlock extends AbstractBlock implements ListBlock
```

Comment:
I guess you removed the wrong one here. Should be ListBLock (otherwise old code searching for ListBLock won't find it).